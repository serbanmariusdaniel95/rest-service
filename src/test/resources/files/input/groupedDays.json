{
  "displayed_what" : "Casa Ferlin",
  "displayed_where" : "Stampfenbachstrasse 38",
  "opening_hours" : {
    "days": {
      "monday": [
        {
          "start": "11:30",
          "end": "14:00",
          "type": "OPEN"
        },
        {
          "start": "18:30",
          "end": "22:00",
          "type": "OPEN"
        }
      ],
      "tuesday": [
        {
          "start": "11:30",
          "end": "14:00",
          "type": "OPEN"
        },
        {
          "start": "18:30",
          "end": "22:00",
          "type": "OPEN"
        }
      ],
      "wednesday": [
        {
          "start": "11:30",
          "end": "14:00",
          "type": "OPEN"
        },
        {
          "start": "18:30",
          "end": "22:00",
          "type": "OPEN"
        }
      ],
      "thursday": [
        {
          "start": "11:30",
          "end": "14:00",
          "type": "OPEN"
        },
        {
          "start": "18:30",
          "end": "22:00",
          "type": "OPEN"
        }
      ],
      "friday": [
        {
          "start": "11:30",
          "end": "14:00",
          "type": "OPEN"
        },
        {
          "start": "18:30",
          "end": "22:00",
          "type": "OPEN"
        }
      ]
    }
  }
}