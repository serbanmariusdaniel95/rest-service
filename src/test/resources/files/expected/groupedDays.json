{
  "name" : "Casa Ferlin",
  "address" : "Stampfenbachstrasse 38",
  "openingHours" : {
    "Monday - Friday": [
      "11:30 - 14:00",
      "18:30 - 22:00"
    ],
    "Saturday - Sunday": [
      "closed"
    ]
  }
}